src/codeflare_sdk/cluster/widgets.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 IBM, Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The widgets sub-module contains the ui widgets created using the ipywidgets package.
+"""
+import ipywidgets as widgets
+import os
+
+
+def cluster_up_down_buttons(appwrapper: bool) -> widgets.Button:
```

Comment:
Can you add the cluster as an arg here and add the onclick functionality

Reply:
Great point Kevin we can remove all Ipywigets mentions from `cluster.py` this way. Updating the PR.